### PR TITLE
Allow running commands at php-fpm entrypoint

### PR DESCRIPTION
**Pull request by hgajjar**

### CODE REVIEW

File: src/_base/helm/app/templates/application/php-fpm/deployment.yaml [hunk 1]
```diff
@@ -31,6 +31,7 @@ spec:
             port: 9000
           initialDelaySeconds: 5
           periodSeconds: 10
+        command: ["/entrypoint.sh"]
```
Comment (kierenevans) [inline src/_base/helm/app/templates/application/php-fpm/deployment.yaml]: Should we put this in the Dockerfile for php-fpm instead? Then only override it in the dynamic part in docker-compose.yml?

Comment (hgajjar) [reply src/_base/helm/app/templates/application/php-fpm/deployment.yaml]: or just set both dynamic and static entrypoints in Dockerfile based on twig condition `{% if @('app.build') == 'static' %}` and remove it completely from docker-compose?

Comment (kierenevans) [reply src/_base/helm/app/templates/application/php-fpm/deployment.yaml]: Yeah as it's now a template - good idea :)

**Review by kierenevans — DISMISSED** [review]

**Comment by hgajjar** [comment]
Hi @kierenevans , could you please rereview this PR?

**Review by kierenevans — DISMISSED** [review]

**Pull request merged**